### PR TITLE
[noticket] Adjust generation

### PR DESCRIPTION
**Pull request by lx-wnk**
- Adjust removal for package
- Adjust examples
- Remove zip generation - leave to ci artifacts
- Switch from black- to allow-list

### CODE REVIEW

File: files/package-plugin.sh [hunk 1]
```diff
@@ -7,21 +7,22 @@ PROJECT_NAME="$1"
...
 done
 
 pushd "${PROJECT_NAME}"
 
 composer install -qno --no-dev --ignore-platform-reqs
 
-if [ ! -f .sw-zip-blacklist ]; then
-    cp -r /tmp/.sw-zip-blacklist.dist .sw-zip-blacklist
+if [ ! -f .sw-zip-allowlist ]; then
+    cp -r /Users/a.wink/scripts/.zip-allowlist .zip-allowlist
```
Comment (kleinmann) [inline files/package-plugin.sh:22]: 🙂

File: files/package-plugin.sh [hunk 2]
```diff
@@ -7,21 +7,22 @@ PROJECT_NAME="$1"
...
 fi
-sed -i -e "s#^#${PROJECT_NAME}/#" .sw-zip-blacklist
+
+comm -2 -3 <(ls -a) <(sort .zip-allowlist) | tail +2 | xargs rm -rf
+rm -rf .zip-allowlist
 
 popd
-zip -rq $PROJECT_NAME.zip $PROJECT_NAME --exclude @$PROJECT_NAME/.sw-zip-blacklist $PROJECT_NAME/.sw-zip-blacklist
```
Comment (kleinmann) [inline files/package-plugin.sh:27]: Why remove the ZIP creation altogether? I like the possibility to download the finished plugin ZIP from the artifacts directly. Maybe both the folder and the ZIP can be kept as artifacts?

File: README.md [hunk 3]
```diff
@@ -2,18 +2,19 @@
 
 Image to zip shopware plugins without any unneeded files
 - Executes `composer install -qno --no-dev --ignore-platform-reqs` on installation to remove dev-dependencies
+- Removes all files not listed in `.zip-allowlist`
 
 ## Usage
 
-### Example .gitlab-ci.yml
+### Example gitlab-ci job
```
Comment (kleinmann) [inline README.md:9]:
```suggestion
### Example GitLab CI job
```

**Review by kleinmann — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jinnoflife) [reply README.md:9]: done

File: files/package-plugin.sh (hunk 1 above)
Comment (jinnoflife) [reply files/package-plugin.sh:22]: done

**Comment by lx-wnk** [comment]
PR is closed due to many possible files like a changelog, which would be excluded by the changes

**Pull request closed without merging**